pkg/jobQueue/jobQueue.go

```diff
@@ -80,11 +80,17 @@ func (q *JobQueue) processRequests() {
 }
 
 func (q *JobQueue) execute(req Request) {
-	log.Debug().Msgf("Executing queue request ID: %s\n", req.ID)
+	log.Debug().
+		Str("request_id", req.ID).
+		Msgf("executing queue request id: %s", req.ID)
```

Comment:
```suggestion
		Msg("executing queue request")
```
the request id is already supplied separately, same for the statements below

Reply:
resolved, should remain in `Msgf`, see [comment](https://github.com/greenbone/opensight-golang-libraries/pull/95#discussion_r1696699393)